source/Engine.cpp

```diff
@@ -1655,20 +1680,23 @@ void Engine::CalculateStep()
 	if(flagship && showFlagship)
 	{
 		DrawShipSprites(*flagship);
-		if(flagship->IsThrusting() && !flagship->EnginePoints().empty())
-		{
-			for(const auto &it : flagship->Attributes().FlareSounds())
-				Audio::Play(it.first);
-		}
-		else if(flagship->IsReversing() && !flagship->ReverseEnginePoints().empty())
+		if (!timePaused)
 		{
-			for(const auto &it : flagship->Attributes().ReverseFlareSounds())
-				Audio::Play(it.first);
-		}
-		if(flagship->IsSteering() && !flagship->SteeringEnginePoints().empty())
-		{
-			for(const auto &it : flagship->Attributes().SteeringFlareSounds())
-				Audio::Play(it.first);
+			if(flagship->IsThrusting() && !flagship->EnginePoints().empty())
+			{
```

Comment:
the `if`s here only contain a single statement, the `for` loop, so don't have parens per coding style.

Reply:
This is also pre-existing code I just moved into an if statement. Can we do a separate PR with code style cleanup in Engine?

Reply:
The style guide says that braces are not required, but allowed (I prefer not to have them). But keep in mind that changing style of pre-existing code in such a large feature PR may delay the merge. So yeah, if you want to change it, do it separately.